dist/flight/flight.controller.d.ts

```diff
@@ -0,0 +1,10 @@
+import { FlightService } from "./flight.service";
+export declare class FlightController {
+    private readonly flightService;
+    constructor(flightService: FlightService);
+    bookFlight(flightFrom: string, flightTo: string, flightDate: string, flightPrice: number, flightSeats: number, flightCompany: string): any;
+    cancelFlight(id: string): any;
```

Comment:
pls avoid using return type as "any"

https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html. - please read it for standard practices